webservices/deployment.md

```diff
@@ -0,0 +1,63 @@
+## Heroku
+
+If you created your app by running `suspenders app_name --heroku true`, your life will be slightly easier when deploying to Heroku.  This flag:
+
+* Creates a staging and production Heroku app
+* Sets them as staging and production Git remotes
+* Configures staging with RACK_ENV and RAILS_ENV environment variables set to staging
+* Adds the Rails Stdout Logging gem to configure the app to log to standard out, which is how Heroku's logging works.
+* Creates a Heroku Pipeline for review apps
+
+(from the [readme](https://github.com/thoughtbot/suspenders))
+
+It also adds the `rails_12factor` gem which is necessary for deploying a Rails app to Heroku.
+
+Deployment in this case is simply a matter of:
+
+(1) Setting all the necessary environment variables, either via the Heroku command line:
+
+  ```
+  heroku config:set ENV_VAR_NAME=value -a name-of-heroku-app
+  ```
+or via the Heroku Dashboard.
+
+(2) Running the deploy script, passing in the environment:
+
+  ```
+  ./bin/deploy staging
+  ```
```

Comment:
Maybe clarify that this has to be run from the root directory of the app, or write out full path like $app-directory/bin/deploy